routingpy/routers/valhalla.py

```diff
@@ -488,6 +484,62 @@ def parse_isochrone_json(response, intervals, locations, interval_type):
 
         return Isochrones(isochrones, response)
 
+    def raster(  # noqa: C901
```

Comment:
it's the same parameters as for isochrones. I'm sure this can be done better to make sure both use the same parameters

Reply:
I agree that it's a good idea to use `/raster`, and we already have a `Raster` class to return the result, so that's all good for me.